src/common/PortForward/ServicePortForwardManager.cs

```diff
@@ -171,7 +171,28 @@ private async Task RunLoopAsync(TcpClient connection, CancellationToken cancella
 
                         while (!requestProcessingCancellationTokenSource.Token.IsCancellationRequested)
                         {
-                            int cRead = await stream.ReadAsync(buffer, 0, buffer.Length, requestProcessingCancellationTokenSource.Token);
+                            int cRead = 0;
+                            try
+                            {
+                                cRead = await stream.ReadAsync(buffer, requestProcessingCancellationTokenSource.Token);
+                            }
+                            catch (IOException ex)
+                            {
+                                if (ex.InnerException is OperationCanceledException)
+                                {
+                                    // Cancellation requested
+                                    break;
+                                }
+                                if (ex.InnerException is SocketException se && se.SocketErrorCode == SocketError.ConnectionReset || se.SocketErrorCode == SocketError.OperationAborted)

```

Comment:
@Eneuman there is build error , can you fix it please ? following runs have failed

https://github.com/Azure/Bridge-To-Kubernetes/actions/runs/5389595610/jobs/9783765958

```/home/runner/work/Bridge-To-Kubernetes/Bridge-To-Kubernetes/src/common/PortForward/ServicePortForwardManager.cs(186,133): error CS0165: Use of unassigned local variable 'se' [/home/runner/work/Bridge-To-Kubernetes/Bridge-To-Kubernetes/src/common/common.csproj::TargetFramework=net6.0]```

Reply:
@hsubramanianaks Fixed!